### ISSUE TITLE
View full hyperparameter details in console output/logging

### ISSUE THREAD

**Issue by cyyeung1234** [keras-team/autokeras#1587]
### Feature Description
Currently, for some reason the "Hyperparameter" column of the console is truncated and I don't see a way to change the output so that this is viewed in full. Look at the snippet below for an example:

### Console Output Example
```
Hyperparameter    |Value             |Best Value So Far 
image_block_1/c...|3                 |3                 
image_block_1/c...|True              |True              
image_block_1/c...|0                 |0                 
image_block_1/c...|2                 |2                 
image_block_1/c...|2                 |2                 
image_block_1/c...|32                |32                
image_block_1/c...|64                |64                
image_block_1/c...|32                |32                
image_block_1/c...|32                |32                
regression_head...|0                 |0                 
optimizer         |adam              |adam              
learning_rate     |2e-05             |0.001  
```
### Reason
It would be a lot more helpful if I can see what these hyperparameter labels are referring to (kernel size, number of filters, etc.).

### Solution
Output the entire hyperparameter column (untruncated) by default.

**Comment by jrhorne** [comment]
Or, and maybe I've missed it, but is there an elegant way to export the optimal hyperparameters found by AutoKeras?

**Comment by cyyeung1234** [comment]
@jrhorne Doing model.summary() is a partial solution but it doesn't show everything.

**Comment by jrhorne** [comment]
Ah, thanks. @cyyeung1234
I think having just a function or something that can export a CSV with all tried hyperparameters, the model's performance, and marking the search progress would be helpful. Being able to see, for example, the top 10 best models all have the same filter size is more helpful than just the top hyperparameters. At least for further refinements.

**Comment by haifeng-jin** [comment]
I think this issue is resolved in KerasTuner already? Please reopen it if it is not resolved.

**Comment by cyyeung1234** [comment]
@haifeng-jin I don't think this is resolved. I updated autokeras and ran the StructuredDataRegressor example today and this issue still exists. 

More generally, **where are the hyperparameter details stored after training**?  I see a bunch of json files but the trail names are gibberish.
